deploy/namespace-operator.yaml

```diff
@@ -1533,14 +1546,59 @@ kind: Service
 metadata:
   labels:
     app: security-profiles-operator
-  name: metrics
+  name: metrics-controller-runtime
+  namespace: security-profiles-operator
+spec:
+  ports:
+  - name: https
+    port: 443
+    targetPort: 8442
+  selector:
+    app: security-profiles-operator
+    name: spod
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: security-profiles-operator
```

Comment:
Should this point to the label for the SPOD? Seems like the spod has a different label.